index.html

```diff
@@ -39,5 +40,189 @@ <h1 id="presentation_text">
             </li>
         </ul>
     </section>
+
+    <!-- Works section -->
+    <section id="recent_works" class="center-content">
+        <div class="scroll_down_btn">
+            <img src="assets/icons/down-arrow.png" alt="scroll down button"/>
+        </div>
+        <h1 class="primary-font-style"><b>My Recent <br> Works</b></h1>
+        <div class="indicator"></div>
+        
+        <div class="works_list">
+        
+            <div class="card">
+            <img src="assets/images/img-placeholder.png" alt="Avatar">
+            <div class="card-footer">
+                <h3 class="project_title">Multi-Post Stories Gain+Glory</h3>
+                <ul class="tags">
+                    <li>Ruby on rails</li>
+                    <li>Css</li>
+                    <li>Javascript</li>
+                    <li>Html</li>
+                </ul> 
+                <div class="action">
```

Comment:
[OPTIONAL] Kindly take note of the corrections in the image below for all your recent works cards.

| Your Design|
|-------------|
|![image](https://user-images.githubusercontent.com/62938735/192089853-ce536d64-a605-4748-aa80-dfd6b6a0fcf0.png)|